src/usr/local/buildpack/utils/ruby.sh

```diff
@@ -4,36 +4,38 @@ function gem_install() {
   tool_path=$(find_tool_path)
 
   if [[ -z "${tool_path}" ]]; then
-    INSTALL_DIR=$(get_install_dir)
-    base_path=${INSTALL_DIR}/${TOOL_NAME}
-    tool_path=${base_path}/${TOOL_VERSION}
+    tool_path=$(create_versioned_tool_path)
+    mkdir -p "${tool_path}"
 
-    mkdir -p ${tool_path}
-
-    GEM_HOME=$tool_path gem install  --bindir $tool_path/bin ${TOOL_NAME} -v ${TOOL_VERSION}
+    GEM_HOME=$tool_path gem install  --bindir "$tool_path"/bin "${TOOL_NAME}" -v "${TOOL_VERSION}"
 
   # TODO: clear gem cache
   fi
 }
 
 
 function gem_link_wrapper() {
-  link_wrapper ${1:-$TOOL_NAME} $tool_path/bin
+  link_wrapper "${1:-$TOOL_NAME}" "$tool_path/bin"
 }
 
 function gem_shell_wrapper () {
-  local install_dir=$(get_install_dir)
-  local FILE="${install_dir}/bin/${1:-$TOOL_NAME}"
+  local install_dir
```

Comment:
I am using [shellcheck](https://github.com/koalaman/shellcheck) for analysis.
In this case it complains about the direct assignment: https://github.com/koalaman/shellcheck/wiki/SC2155

I can change it back and we can introduce shellcheck suggestions later if you want or ignore it alltogether.

Reply:
No, leave it. Let's add spell check here. We have it in renovate docs repo.

Can we easily add it before other PR to this repo with ci checks? Or is it too complicated?

Reply:
We can do that, but should then restrict it to changed files only so that we have shellcheck in place going further.

AS we have quite a lot of shell code, shellcheck also has quite a bit to complain.

Reply:
Ok, so let's do this first and then spell check